modules/ROOT/pages/deployment/services/caching.adoc

```diff
@@ -0,0 +1,46 @@
+= Caching
+:toc: right
+:description: Setup caching can not only be a big boost on performance but can also be essential when setting up Infinite Scale via container orchestration.
+
+== Introduction
+
+{description} Caching is also recommended for some services like to persist event data for later processing, or, not limited to but especially when the underlaying services are scaled via xref:deployment/container/orchestration/orchestration.adoc[container orchestration].
+
+== General Information
+
+These are the main aspects that require caching::
++
+--
+* *Performance*: This kind of cache does not need to be persistent and in case of outages, the cache gets rebuild and no information is lost. This is typically used for caching metadata and filesystem access.
```

Comment:
```suggestion
* *Performance*: This kind of cache does not need to be persistent and in case of outages, the cache is rebuilt and no information is lost. This is typically used for caching metadata and filesystem access.
```